src/Servers/Kestrel/Transport.Libuv/src/Internal/LibuvConnectionListener.cs

```diff
@@ -147,6 +147,7 @@ internal async Task BindAsync()
                 {
                     var listener = new Listener(TransportContext);
                     _listeners.Add(listener);
+                    listener.OnShutdown = OnListenerShutdown;
```

Comment:
Why add an OnShutdown callback just to log? You can make the same log from LibuvThread.

Reply:
Makes sense. I'll make the change.